metabench.cmake

```diff
@@ -274,11 +274,11 @@ file(WRITE ${METABENCH_RB_PATH}
 ################################################################################
 set(MEASURE_RB_PATH ${METABENCH_DIR}/measure.rb)
 file(WRITE ${MEASURE_RB_PATH}
-"require 'benchmark'                                                    \n"
-"command = ARGV.join(' ')                                               \n"
-"time = Benchmark.realtime { `#{command}` }                             \n"
-"puts \"[command line: #{command}]\"                                    \n"
-"puts \"[compilation time: #{time}]\"                                   \n"
+"require 'benchmark'                                                         \n"
+"command = ARGV.map{ |arg| arg.match(/\\s/) ? '\"'+arg+'\"' : arg }.join(' ')\n"
```

Comment:
I know this works, but it looks like something is wrong. The command line we receive in the compiler launcher is what CMake gave us; that command line should not need any modification it seems.